src/BenchmarksClient/Workers/WrkWorker.cs

```diff
@@ -233,6 +233,9 @@ private static Process StartProcess(ClientJob job)
                 var p99Match = Regex.Match(job.Output, String.Format(LatencyPattern, "99%"));
                 job.Latency.Within99thPercentile = ReadLatency(p99Match);
 
+                var p100Match = Regex.Match(job.Output, @"\s+Latency\s+[\d\.]+\w+\s+[\d\.]+\w+\s+([\d\.]+)(\w+)");
```

Comment:
Did you try it?

Reply:
On an online regex visualizer with an example of wrk's output

Reply:
I also tested it, it works